exonum/src/node/consensus.rs

```diff
@@ -117,29 +117,25 @@ impl NodeHandler {
             return;
         }
 
+        trace!("Handle propose");
+
         let snapshot = self.blockchain.snapshot();
-        // Check that transactions are not committed yet
-        for hash in msg.transactions() {
-            if Schema::new(&snapshot).transactions().contains(hash) {
-                error!(
-                    "Received propose with already committed transaction, msg={:?}",
-                    msg
-                );
+        let schema = Schema::new(&*snapshot);
+        //TODO: remove this match after errors refactor.
```

Comment:
I propose to remove this "TODO".

Reply:
`handle_propose` still can't return error 😢

Reply:
let's mention here related issue